### DISCUSSION TITLE
Using ArchR for bulk ATAC-seq data

### DISCUSSION THREAD

**Discussion by nandankita** [GreenleafLab/ArchR#1246] [Questions / Documentation]
I would like to use **addReproduciblePeakSet** function for my bulk ATAC data. In order to do that I tried creating arrow files, but ended up with error 

```
<simpleError: Detected 2 or less cells (0 barcodes have greater than 50 fragment
s) in file! Check inputs such as 'minFrags' or 'maxFrags' to keep cells! Exiting!>
```

Would you please mention how can I use ArchR for bulk data? Do I even need to create arrow files/ ArchR project? If yes, what will be the parameters for **createArrowFiles** function? 

Would you have any example for me to follow?

**Comment by 1221li** [comment]
Hello, I have encountered the same issue, I would like to ask, have you solved this problem？I'm looking forward to your help！

**Reply by nandankita** [reply]
Unfortunately, I did NOT find a solution for this error. Are you also trying for bulk ATAC data?

I wanted to ultimately call consensus for my bulk data using ArchR addReproduciblePeakSet so I just modified that function to use bulk data directly instead of using arrow files.

**Reply by rcorces** [reply]
As stated on the website, ArchR is for single-cell analysis. It is not intended for bulk ATAC-seq data

**Reply by 1221li** [reply]
In fact, my purpose is to analyze single-cell ATAC and bulk ATAC together to find the difference peaks between these two ATACS. I would like to ask whether ARCHR can meet such requirements.

**Reply by rcorces** [reply]
Unfortunately, ArchR cannot handle Bulk ATAC-seq data. This may happen one day but not any time soon.
